src/main/webapp/WEB-INF/views/users.jsp

```diff
@@ -84,33 +87,33 @@
 <br/><br/>
 <%--<h4><spring:message code="lang.addNewUser"/></h4>--%>
 <%--<form action="<%=EndPoints.USER%>" method="post">--%>
-    <%--<input type="email" name="email"/> <spring:message code="lang.email"/><br/>--%>
-    <%--<input type="password" name="password"/> <spring:message code="lang.password"/><br/>--%>
-    <%--<input type="text" name="isAdmin"/> <spring:message code="lang.isAdmin"/><br/>--%>
-    <%--<input type="text" name="firstName"/> <spring:message code="lang.firstName"/> <br/>--%>
-    <%--<input type="text" name="lastName"/> <spring:message code="lang.lastName"/> <br/>--%>
-    <%--<input type="text" name="documentInfo"/> <spring:message code="lang.document"/> <br/>--%>
-    <%--<input type="datetime-local" name="birthday"/> <spring:message code="lang.birthday"/> <br/>--%>
-    <%--<input type="submit" value="OK"/>--%>
+<%--<input type="email" name="email"/> <spring:message code="lang.email"/><br/>--%>
+<%--<input type="password" name="password"/> <spring:message code="lang.password"/><br/>--%>
```

Comment:
removed all comments, why you save them ?

Reply:
I deleted